examples/tests/valid/capture_tokens.test.w

```diff
@@ -4,7 +4,7 @@ class MyResource {
   api: cloud.Api;
   url: str;
 
-  extern "./url_utils.js" pub static inflight isValidUrl(url: str): bool;
+  extern "./url_utils.ts" pub static inflight isValidUrl(url: str): bool;
```

Comment:
@MarkMcCulloh Just checking, so we've decided to officially support *.ts files? Should we note this in the documentation or language spec somewhere? (I'm cool with the choice, just wanna check if we're tracking it somewhere :^))